packages/riipen-ui/src/components/RadioButtonGroup.jsx

```diff
@@ -61,7 +61,7 @@ const RadioButtonGroup = ({
   };
 
   const childrenWithProps = React.Children.map(
-    children,
+    children || [],
```

Comment:
Looks like children is a required prop, so if we want to do this (also maybe as a default prop?), should update the prop type